src/libcore/cell.rs

```diff
@@ -512,6 +514,7 @@ impl<T: ?Sized> Cell<T> {
     #[inline]
     #[stable(feature = "as_cell", since = "1.37.0")]
     pub fn from_mut(t: &mut T) -> &Cell<T> {
+        // SAFETY: `&mut` ensures unique access
```

Comment:
General style note (not hugely important but useful for the future): when what you are writing is a proper sentence, please end it in a `.`.